package.json

```diff
@@ -162,7 +162,7 @@
     "express": "^4.16.4",
     "fs-extra": "^10.1.0",
     "glob": "^7.1.2",
-    "husky": "^1.2.0",
+    "husky": "^8.0.1",
```

Comment:
You also need to add a `prepare` script in order to register the githooks.

https://typicode.github.io/husky/#/?id=manual

```json
{
  "scripts": {
    "prepare": "husky install"
  }
}
```


You should also be able to drop the dependency to `husky` if you add this to the prepare script.

```json
{
  "prepare": "git config core.hookspath .githooks"
}
```

The githook itself then looks like this

```
#!/usr/bin/env sh

npx lint-staged
```

Reply:
Yes, you are right.

Let's stick with `husky` for now.